src/extension.ts

```diff
@@ -69,6 +69,7 @@ export function activate(context: vscode.ExtensionContext): void {
 
     const actionHandler: AzureActionHandler = new AzureActionHandler(context, outputChannel, reporter);
     actionHandler.registerCommand('appService.Refresh', async (node?: IAzureNode) => await tree.refresh(node));
+    vscode.commands.registerCommand('azureStorage.selectSubscriptions', () => vscode.commands.executeCommand("azure-account.selectSubscriptions"));
```

Comment:
appService.selectSubscriptions 

And ditto what Nathan said - you have to add this to package.json